lib/src/components/NxTreeViewSelect/AbstractTreeViewSelect.tsx

```diff
@@ -5,124 +5,15 @@
-
-function AbstractTreeViewSelect<T extends Option>(props: Props<T>) {
-
-  const {
-        options,
-        onToggleCollapse,
-        filterPlaceholder,
-        children,
-        id: idProp,
-        filter,
-        onFilterChange,
-        disabledTooltip,
-        optionTooltipGenerator,
-        tooltipModifierClass,
-        name,
-        renderOption,
-        renderToggleAllOption,
-        renderCounter
-      } = props,
-      filterThreshold = props.filterThreshold || 10,
-      isOpen = !!props.isOpen,
-      disabled = !!props.disabled || !options.length,
-      id = useUniqueId('nx-tree-view-select', idProp || undefined),
-      filteredOptions = props.filteredOptions || options;
-
-  function getTooltipProps(title: string) {
-
-    const tooltipProps: TooltipConfigProps = {
-      title,
-      placement: 'top'
-    };
-
-    if (tooltipModifierClass) {
-      tooltipProps.className = tooltipModifierClass;
-    }
-    return tooltipProps;
-  }
-
-  const renderedOptions = filteredOptions.map((item: T) => {
-    const option = renderOption(item),
-        key = `key-${item.id}`;
-
-    return optionTooltipGenerator ? (
-      <NxTooltip key={key} {...getTooltipProps(optionTooltipGenerator(item))}>
-        <NxTreeViewChild>{option}</NxTreeViewChild>
-      </NxTooltip>
-    ) : <NxTreeViewChild key={key}>{option}</NxTreeViewChild>;
-  });
-
-  const wrappedTriggerContent = ensureElement(children);
-
-  const counter = renderCounter && renderCounter();
-
-  const triggerWithCounter = counter ? (
-    <>
-      {wrappedTriggerContent}
-      {counter}
-    </>
-  ) : wrappedTriggerContent;
-
-  const filterContent = onFilterChange && options.length > filterThreshold && (
-    <NxFilterInput disabled={disabled}
-                   placeholder={filterPlaceholder || 'filter'}
-                   id={generateId(name, 'filter-input')}
-                   aria-controls={id}
-                   onChange={onFilterChange}
-                   value={filter || ''}/>
-  );
-
-  const selectAllOption = renderToggleAllOption && renderToggleAllOption();
-
-  const getTriggerTooltip = () => {
-    if (disabled) {
-      if (disabledTooltip) {
-        return getTooltipProps(disabledTooltip);
-      }
-      else if (!options.length) {
-        return getTooltipProps(`There are no ${name} options`);
-      }
-    }
-    return null;
-  };
-
-  return (
-    <NxTreeView onToggleCollapse={onToggleCollapse}
-                isOpen={isOpen && options.length > 0}
-                id={id}
-                triggerContent={triggerWithCounter}
-                triggerTooltip={getTriggerTooltip()}
-                disabled={disabled}
-                className="nx-tree-view--select"
-                aria-describedby={counter && counter.props.id}>
-      {filterContent}
-      { selectAllOption && <NxTreeViewChild>{selectAllOption}</NxTreeViewChild> }
-      {renderedOptions}
-    </NxTreeView>
-  );
-}
-
-export default AbstractTreeViewSelect;
+export {
```

Comment:
Are any of the exports in this file still needed?

Reply:
> Are any of the exports in this file still needed?

Removed: a431a5623b68c916560e31a0431e53edb5a634ac